test/swimsuit.js

```diff
@@ -38,4 +38,17 @@ describe('Swimsuit', () => {
 
     expect(Swimsuit.test instanceof Promise).to.be.true
   })
+
+  it('should add support with a promise', (done) => {
+    const returnVal = 'Hello'
+    Swimsuit.supports('wait', () => {
+      return new Promise((resolve, reject) => {
```

Comment:
Does standard not allow single-statement arrow functions without the `{ ... }` anymore?